app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html

```diff
@@ -103,6 +103,4 @@ <h1 class="govuk-heading-l">@messages("dashboard.unknown.h1")</h1>
     <p class="govuk-body">@Html(messages("dashboard.unknown.p2", gvmsLink))</p>
 }
 
-@gvmsLink = {
-    <a id="gvms_url" href="https://www.gov.uk/guidance/get-a-goods-movement-reference#get-a-goods-movement-reference" class="govuk-link govuk-link--no-visited-state">@messages("dashboard.gvms.link_text")</a>
-}
+@gvmsLink = {<a id="gvms_url" href="https://www.gov.uk/guidance/get-a-goods-movement-reference#get-a-goods-movement-reference" class="govuk-link govuk-link--no-visited-state">@messages("dashboard.gvms.link_text")</a>}
```

Comment:
Can you define the message key for the link text in terms of the serviceId, use a generic name for the link instead of `gvmsLink` and, for the time being, allow the link href to be supplied as a parameter to the view?

Reply:
Good idea. Done it. Mainly here: 

https://github.com/hmrc/customs-service-status-frontend/pull/19/commits/985456e8526f42488cfe81ec42fa29c1ddd3bf51#diff-cee4d30ef8f4d0eba8ebba997e13782fc0fb776a105b9fe226e47e9c60d2ca19R106